src/background/utils/overwriteWitnetNodeConfiguration.ts

```diff
@@ -14,17 +14,20 @@ export function overwriteWitnetNodeConfiguration({
   try {
     fs.writeFileSync(
       path.join(sheikahPath, witnetConfigFileName),
-
       fs
         .readFileSync(path.join(sheikahPath, witnetConfigFileName))
         .toString()
         .replace(
-          'node_url = "127.0.0.1:21338"',
+          /node_url =([^;]*)"/,
           `node_url = ${JSON.stringify(publicNodeUrls)}\n`.replace("'", ''),
         )
-        .replace('52.166.178.145:21338', `20.126.70.77:21338`),
+        .replace(
+          /node_url =([^;]*)]/,
+          `node_url = ${JSON.stringify(publicNodeUrls)}\n`.replace("'", ''),
+        ),
```

Comment:
Isn't this exactly the same as the previous replace?

```suggestion
```

Reply:
It is not, the first one replace the node urls within `""` and the second one within `[]`